crates/miden-tx/src/host/mod.rs

```diff
@@ -738,28 +767,13 @@ where
         }
 
         // get the KEY to which the slot is being updated
-        let key = Word::new([
-            process.get_stack_item(4),
-            process.get_stack_item(3),
-            process.get_stack_item(2),
-            process.get_stack_item(1),
-        ]);
+        let key = process.get_stack_word_be(1 + 1);
 
         // get the previous VALUE of the slot
-        let prev_map_value = Word::new([
-            process.get_stack_item(8),
-            process.get_stack_item(7),
-            process.get_stack_item(6),
-            process.get_stack_item(5),
-        ]);
+        let prev_map_value = process.get_stack_word_be(5 + 1);
 
         // get the VALUE to which the slot is being updated
-        let new_map_value = Word::new([
-            process.get_stack_item(12),
-            process.get_stack_item(11),
-            process.get_stack_item(10),
-            process.get_stack_item(9),
-        ]);
+        let new_map_value = process.get_stack_word_be(9 + 1);
```

Comment:
```suggestion
        let new_map_value = process.get_stack_word(9 + 1);
```

Replacing these with the original `get_stack_word` seems to work, they also return the BE words. Is the extension trait then still needed?